pymc3/plots.py

```diff
@@ -110,17 +111,21 @@ def traceplot(trace, varnames=None, transform=lambda x: x, figsize=None,
 
 
 def histplot_op(ax, data, alpha=.35):
+    colors = []
     for i in range(data.shape[1]):
         d = data[:, i]
 
         mind = np.min(d)
         maxd = np.max(d)
         step = max((maxd - mind) // 100, 1)
-        ax.hist(d, bins=range(mind, maxd + 2, step), alpha=alpha, align='left')
+        h = ax.hist(d, bins=range(mind, maxd + 2, step), alpha=alpha, align='left')
+        colors.append(h[2][0].get_facecolor())
         ax.set_xlim(mind - .5, maxd + .5)
+    return colors
```

Comment:
Not sure I like the API here, returning just colors is pretty uncommon. Could we instead return the axis handle and then retrieve the colors from there in `traceplot`?